centos/centos-7.x/common/install_utils.sh

```diff
@@ -30,14 +30,29 @@ yum install -y numactl \
     cmake \
     libnl3-devel \
     libsecret \
-    https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/d/dkms-3.0.5-1.el7.noarch.rpm \
     rpm-build \
     make \
     check \
     check-devel \
     subunit \
-    subunit-devel
-    
+    subunit-devel \
+    git
+
+## Install dkms from the EPEL repository
+wget -r --no-parent -A "dkms-*.el7.noarch.rpm" https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/d/ 
+yum localinstall ./dl.fedoraproject.org/pub/epel/7/x86_64/Packages/d/dkms-*.el7.noarch.rpm -y
+
+## Install jq Utility
+# Download dependency libonig.so for jq
+wget -r --no-parent -A "oniguruma-*.el7.x86_64.rpm" https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/o/
+yum localinstall ./dl.fedoraproject.org/pub/epel/7/x86_64/Packages/o/oniguruma-*.el7.x86_64.rpm -y
+# Download jq utility
+wget -r --no-parent -A "jq-*.el7.x86_64.rpm" https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/j/
+yum localinstall ./dl.fedoraproject.org/pub/epel/7/x86_64/Packages/j/jq-*.el7.x86_64.rpm -y
```

Comment:
sorry, I did not make it clearly. I meant is it possible to do just `yum install jq`?
If we need to add other yum repos to do so, is it worth? This way might be easier if we want to install other packages in the future, and we won't need to download the rpm every time.

Reply:
jq is a part of EPEL repository. We didn't want include this repository in our images and so we had to download only the specific packages. 
With the above structure, the changes to the versions in the repository are already taken care of.